bitnami/concourse/templates/web/ingress.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.web.enabled -}}
```

Comment:
If we're using ingress then we should also have the tls-secrets.yaml file that we have in other charts.